Liquibase GraalVM Failure 22.2

https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424

```
Error starting Micronaut server: Bean definition [javax.sql.DataSource] could not be loaded: java.lang.NoSuchMethodException: liquibase.configuration.ConfiguredValueModifierFactory.<init>()
[65](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L65)io.micronaut.context.exceptions.BeanInstantiationException: Bean definition [javax.sql.DataSource] could not be loaded: java.lang.NoSuchMethodException: liquibase.configuration.ConfiguredValueModifierFactory.<init>()
[66](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L66)	at io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1915)
[67](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L67)	at io.micronaut.context.DefaultApplicationContext.initializeContext(DefaultApplicationContext.java:245)
[68](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L68)	at io.micronaut.context.DefaultBeanContext.readAllBeanDefinitionClasses(DefaultBeanContext.java:3320)
[69](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L69)	at io.micronaut.context.DefaultBeanContext.finalizeConfiguration(DefaultBeanContext.java:3678)
[70](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L70)	at io.micronaut.context.DefaultBeanContext.start(DefaultBeanContext.java:341)
[71](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L71)	at io.micronaut.context.DefaultApplicationContext.start(DefaultApplicationContext.java:190)
[72](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L72)	at io.micronaut.runtime.Micronaut.start(Micronaut.java:75)
[73](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L73)	at io.micronaut.runtime.Micronaut.run(Micronaut.java:323)
[74](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L74)	at io.micronaut.runtime.Micronaut.run(Micronaut.java:309)
[75](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L75)	at micronaut.example.Application.main(Application.java:13)
[76](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L76)Caused by: liquibase.exception.UnexpectedLiquibaseException: java.lang.NoSuchMethodException: liquibase.configuration.ConfiguredValueModifierFactory.<init>()
[77](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L77)	at liquibase.Scope.getSingleton(Scope.java:329)
[78](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L78)	at liquibase.configuration.LiquibaseConfiguration.getCurrentConfiguredValue(LiquibaseConfiguration.java:130)
[79](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L79)	at liquibase.configuration.ConfigurationDefinition.getCurrentConfiguredValue(ConfigurationDefinition.java:93)
[80](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L80)	at liquibase.configuration.ConfigurationDefinition.getCurrentValue(ConfigurationDefinition.java:61)
[81](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L81)	at liquibase.database.AbstractJdbcDatabase.getDatabaseChangeLogLockTableName(AbstractJdbcDatabase.java:664)
[82](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L82)	at liquibase.changelog.ChangeLogParameters.<init>(ChangeLogParameters.java:59)
[83](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L83)	at liquibase.Liquibase.<init>(Liquibase.java:117)
[84](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L84)	at io.micronaut.liquibase.AbstractLiquibaseMigration.createLiquibase(AbstractLiquibaseMigration.java:225)
[85](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L85)	at io.micronaut.liquibase.AbstractLiquibaseMigration.migrate(AbstractLiquibaseMigration.java:138)
[86](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L86)	at io.micronaut.liquibase.AbstractLiquibaseMigration.forceRun(AbstractLiquibaseMigration.java:100)
[87](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L87)	at io.micronaut.liquibase.AbstractLiquibaseMigration.run(AbstractLiquibaseMigration.java:86)
[88](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L88)	at io.micronaut.liquibase.LiquibaseMigrationRunner.lambda$onCreated$1(LiquibaseMigrationRunner.java:64)
[89](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L89)	at java.util.Optional.ifPresent(Optional.java:183)
[90](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L90)	at io.micronaut.liquibase.LiquibaseMigrationRunner.onCreated(LiquibaseMigrationRunner.java:62)
[91](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L91)	at io.micronaut.liquibase.LiquibaseMigrationRunner.onCreated(LiquibaseMigrationRunner.java:39)
[92](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L92)	at io.micronaut.context.DefaultBeanContext.triggerBeanCreatedEventListener(DefaultBeanContext.java:2401)
[93](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L93)	at io.micronaut.context.DefaultBeanContext.postBeanCreated(DefaultBeanContext.java:2379)
[94](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L94)	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2303)
[95](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L95)	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2245)
[96](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L96)	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3010)
[97](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L97)	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
[98](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L98)	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:2912)
[99](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L99)	at io.micronaut.context.DefaultBeanContext.loadContextScopeBean(DefaultBeanContext.java:2731)
[100](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L100)	at io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1909)
[101](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L101)	... 9 common frames omitted
[102](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L102)Caused by: java.lang.NoSuchMethodException: liquibase.configuration.ConfiguredValueModifierFactory.<init>()
[103](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L103)	at java.lang.Class.getConstructor0(DynamicHub.java:3349)
[104](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L104)	at java.lang.Class.getDeclaredConstructor(DynamicHub.java:2553)
[105](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L105)	at liquibase.Scope.getSingleton(Scope.java:324)
[106](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L106)	... 32 common frames omitted
[107](https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/jobs/2805640424#L107)+ curl -s localhost:8080/users`
```

--- a/liquibase/src/main/resources/META-INF/native-image/io.micronaut.liquibase/micronaut-liquibase/reflect-config.json
+++ b/liquibase/src/main/resources/META-INF/native-image/io.micronaut.liquibase/micronaut-liquibase/reflect-config.json
@@ -16,6 +16,16 @@
     "methods":[{"name":"<init>","parameterTypes":[] }]
   },
   {
+    "name":"liquibase.Scope",
+    "methods":[{"name":"<init>","parameterTypes":[] }]
+  },
+  {
+    "name":"liquibase.configuration.ConfiguredValueModifierFactory",
+    "allDeclaredFields":true,
+    "allPublicMethods":true,
+    "methods":[{"name":"<init>","parameterTypes":[] }]
+  },
+  {
     "name":"liquibase.AbstractExtensibleObject",
     "allPublicMethods":true
   },

